frontend/i18n.config.ts

```diff
@@ -2,4 +2,12 @@ export default defineI18nConfig(() => ({
   locale: 'en',
   fallbackLocale: 'en',
   objectNotation: true,
+  missing: (locale, key, vm) => {
+    // TODO(grady) figure out how to skip this if we're in production + just log.
```

Comment:
If `i18n.config.ts` is loaded at runtime (which I'm guessing it is, based on this), you should be able to use something like `process.env.NODE_ENV == 'prod'` or something, and then set those in our `package.json` scripts

Reply:
Good suggestion - done.